platform/core/src/classes/hotkeys/index.js

```diff
@@ -1,17 +1,16 @@
 // Only imported in environment w/ `window`
 // So we need to mock these for tests
 import Mousetrap from 'mousetrap';
-import pausePlugin from 'mousetrap/plugins/pause/mousetrap-pause.js';
-import recordPlugin from 'mousetrap/plugins/record/mousetrap-record.js';
+import pausePlugin from './pausePlugin';
+import recordPlugin from './recordPlugin';
 
-// import pausePlugin from './pausePlugin.js';
-// import recordPlugin from './recordPlugin.js';
+Mousetrap.pause = pausePlugin;
 
-// // // TODO: May need to bind these so Mousetrap = this in plugins;
-// pausePlugin(Mousetrap);
-// recordPlugin(Mousetrap);
-
-// console.log(Mousetrap);
-// console.log(Object.keys(Mousetrap));
+// Initialize record plugin
+// https://github.com/ccampbell/mousetrap/blob/master/plugins/record/mousetrap-record.js#L189-L201
+//
```

Comment:
This comment is necessary? maybe adding a TODO if there's something to be done in the future?

Reply:
I've changed a bit the comments, It was probably added because the `MouseTrap` plugin is initialyzed by itself, so its just a headsup.